### ISSUE TITLE
[iOS 15] tintColor not applied to Image view on React Native 62.2

### ISSUE THREAD

**Issue by isinha-onefc** [facebook/react-native#32251]
## Description
I have some code like this throughout my app:

    <Image
      style={{width: 50, height: 50, tintColor: 'yellow'}}
      source={fabicon}
    />

This is working fine on Android and on iOS <15. However, the `tintColor` is not applied to any of my icons on iOS 15.

I am on React Native version 0.62.2.

Do I need to upgrade to the latest version to apply tint color to images on iOS 15 (and above)?

## React Native version:
0.62.2

**Comment by mconiglio** [comment]
I'm seeing the same issue

**Comment by fydxmy** [comment]
At the time of development, part of the network picture can not be displayed, also do not report errors

**Comment by hesselbom** [comment]
Upgraded to react-native version 0.65.1 but I still get the same issue in iOS 15. I got a similar issue in a separate SwiftUI project in iOS 15 (where I had to use SwiftUI native way of doing things with foregroundColor instead of hacking UIKit) so I'm guessing iOS 15 changed something with how colors are set.

**Comment by Alex94800** [comment]
Same issue here, iOS 15 and react-native 0.65.1.

**Comment by hesselbom** [comment]
If I create a brand new blank project with `create-react-native-app` and add an <Image> component with tintColor set in styles it works. So there's something else in my project at least that interferes with the Image's tintColor. I'll try to create a minimal, reproducible example when I find what that is.

**Comment by EndlezzCoding** [comment]
I noticed if I change the tintColor several times through set state, the color would eventually change. 
It seems that it does not work when setting the tintColor first time it renders.

**Comment by isinha-onefc** [comment]
> If I create a brand new blank project with create-react-native-app and add an  component with tintColor set in styles it works. So there's something else in my project at least that interferes with the Image's tintColor. I'll try to create a minimal, reproducible example when I find what that is.

If I am not wrong, this is for an expo project. Have you tried with `react-native init` as well?

**Comment by isinha-onefc** [comment]
> I noticed if I change the tintColor several times through set state, the color would eventually change. It seems that it does not work when setting the tintColor first time it renders.

Yes, same here. But quitting the screen and coming back to it removes the tint color again.

**Comment by hesselbom** [comment]
> > If I create a brand new blank project with create-react-native-app and add an  component with tintColor set in styles it works. So there's something else in my project at least that interferes with the Image's tintColor. I'll try to create a minimal, reproducible example when I find what that is.
> 
> If I am not wrong, this is for an expo project. Have you tried with `react-native init` as well?

I have. I've also now tried with latest version of `react-native-navigation` which I thought was the culprit but was unable to reproduce it. Have you been able to create a minimal reproducible example?

**Comment by YourBoB** [comment]
Having same issue, use 0.64.2

**Comment by Alex94800** [comment]
Any news on this ? Very annoying, I can't find the issue.

**Comment by Alex94800** [comment]
Guys, I found it ! I has this patch I added because images did not load correctly on iOS 14. https://github.com/facebook/react-native/issues/29279#issuecomment-657201709.

I removed it, updated to RN 0.66, and everything now works perfectly !

**Comment by EndlezzCoding** [comment]
> Guys, I found it ! I has this patch I added because images did not load correctly on iOS 14. https://github.com/facebook/react-native/issues/29279#issuecomment-657201709.
> 
> 
> 
> I removed it, updated to RN 0.66, and everything now works perfectly !

Great findings, I can confirm the iOS 14 image fix caused the issue.

**Comment by hesselbom** [comment]
> Guys, I found it ! I has this patch I added because images did not load correctly on iOS 14. [#29279 (comment)](https://github.com/facebook/react-native/issues/29279#issuecomment-657201709).
> 
> I removed it, updated to RN 0.66, and everything now works perfectly !

Wow, that was it for me as well. If that was @isinha-onefc 's issue too I think we could close this issue.

**Comment by blaze-dot** [comment]
> I noticed if I change the tintColor several times through set state, the color would eventually change.
> It seems that it does not work when setting the tintColor first time it renders.

I'm seeing the similar behavior. It doesn't render correctly when first presented but renders after I switch to another tab and then back. 

It renders correctly prior to iOS 15.

I'm currently on RN 61.5 iOS 15.

**Comment by tarouboy** [comment]
If anyone want a quick workaround. 
I am using [react-native-fast-image](https://github.com/DylanVann/react-native-fast-image) for all tinted images and it works fine on iOS 15.

**Comment by isinha-onefc** [comment]
I confirm that Image `tintColor` works in iOS 15 on RN 0.66.0

**Comment by hppppong** [comment]
if you are using
https://github.com/facebook/react-native/issues/29279#issuecomment-657201709 
try this
https://github.com/facebook/react-native/issues/29279#issuecomment-843791638

they are different

**Comment by ecemac** [comment]
I'm still having this problem on RN `0.66.4`

**Comment by ronastlelobo** [comment]
@ecemac same here. It doesn't work in `0.66.4`. Any help ?
